src/test/run-pass/issues/issue-62350.rs

```diff
@@ -0,0 +1,46 @@
+// run-pass
```

Comment:
How is this test different from code added to `abi-sysv64-arg-passing`?

Reply:
That test contains duplicates of all the SysV x64 tests, this is the real test.